### PR TITLE
feat: relayer refactor and cleanup + relayer setupscripts

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
- expects LOOK_UP_TABLE env var to be set at all times when starting relayer
local/tests: overwrites env var with pre-loaded lookuptable (same key for everyone)
public networks: expects dev that hosts relayer to init a valid lookuptable and set env var accordingly. 
- this is what the `createNewLookUpTable.ts` script is for
same for the rest of the relayer-setup:
1) relayer accounts get funded only on local /test setup
2) public networks expect the keys provided by developer (env) to be initialized and funded. they can use the new `fundRelayer` script for this.

also:
- introduces standardized error codes for relayer directory
- removes other redundant code
- renaming env vars for clarity

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,7 +12,8 @@
 */light-test-cache/
 /**/.DS_store
 /cli/bin
-
+*/light-anchor 
+light-anchor
```
Comment (vadorovsky) [inline .gitignore:16]: Why do we need this? Install.sh script installs light-anchor into .local/, which is already in gitignore

**Review by ananas-block — COMMENTED** [review]
Second Michals remark, otherwise looks good to me.

File: .gitignore (hunk 1 above)
Comment (SwenSchaeferjohann) [reply .gitignore]: good catch this is not needed

**Comment by SwenSchaeferjohann** [comment]
thx for the reviews. addressed your comment.

I had some more changes to docker and therefore split this into a dedicated PR which can be merged after this one here: #331. feel free to do a final sanity check and merge

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**